src/extension.ts

```diff
@@ -449,37 +443,37 @@ export class Ext extends Ecs.System<ExtEvent> {
         return Rect.Rectangle.from_meta(meta);
     }
 
-    on_active_hint() {
-        if (this.settings.active_hint()) {
-            this.active_hint = new active_hint.ActiveHint(this.dpi);
+    // on_active_hint() {
+    //     if (this.settings.active_hint()) {
+    //         this.active_hint = new active_hint.ActiveHint(this.dpi);
 
-            const focused = this.focus_window();
-            if (focused) {
-                this.active_hint.track(focused);
-            }
-        } else if (this.active_hint) {
-            this.active_hint.destroy();
-            this.active_hint = null;
-        }
-    }
+    //         const focused = this.focus_window();
+    //         if (focused) {
+    //             this.active_hint.track(focused);
+    //         }
+    //     } else if (this.active_hint) {
+    //         this.active_hint.destroy();
+    //         this.active_hint = null;
+    //     }
+    // }
 
     on_active_workspace_changed() {
-        const refocus_hint = () => {
-            if (!this.active_hint?.tracked) return;
-            let active = this.windows.get(this.active_hint.tracked.entity);
-            if (!active) return;
-
-            let aws = this.workspace_id(active);
-            let cws = this.workspace_id(null);
-
-            if (aws[0] === cws[0] && aws[1] === cws[1]) {
-                this.active_hint.track(active);
-            } else {
-                this.active_hint.untrack();
-            }
-        };
-
-        refocus_hint();
+        // const refocus_hint = () => {
+        //     if (!this.active_hint?.tracked) return;
+        //     let active = this.windows.get(this.active_hint.tracked.entity);
+        //     if (!active) return;
+
+        //     let aws = this.workspace_id(active);
+        //     let cws = this.workspace_id(null);
+
+        //     if (aws[0] === cws[0] && aws[1] === cws[1]) {
+        //         this.active_hint.track(active);
+        //     } else {
+        //         this.active_hint.untrack();
+        //     }
+        // };
+
+        // refocus_hint();
```

Comment:
We can remove all code that has been commented out.

Reply:
Removed all outdated code.